SplunkAppForWazuh/appserver/static/css/styles/component.css

```diff
@@ -430,6 +430,10 @@ border-bottom: 1px solid;
   margin-right: 10px;
 }
 
+.wz-margin-top-10 {
+  margin-top: 10px;
+}
```

Comment:
Duplicated class. It's already on `wazuh_decoration.css`

https://github.com/wazuh/wazuh-splunk/blob/223c677991ba311293dd6b5a716eccba919b16ca/SplunkAppForWazuh/appserver/static/css/styles/wazuh_decorations.css#L422